src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/jpa/Visit.kt

```diff
@@ -23,16 +21,14 @@ import javax.persistence.TemporalType
 @Entity
 @Table(name = "VISIT")
 data class Visit(
+
   @Id
-  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "visit_seq_generator")
-  @GenericGenerator(
-    name = "visit_seq_generator",
-    strategy = "uk.gov.justice.digital.hmpps.visitscheduler.jpa.generator.ReferenceIdGenerator",
-    parameters = [
-      Parameter(name = org.hibernate.id.enhanced.SequenceStyleGenerator.SEQUENCE_PARAM, value = "visit_seq"),
-    ]
-  )
-  val id: String = "",
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "ID")
+  val id: Long = 0,
+
+  @Column(name = "REFERENCE")
```

Comment:
Consider @NaturalId

Reply:
Just had a look into this and it makes it An immutable therefore we can't use it